packages/babel-preset-flow/README.md

```diff
@@ -0,0 +1,53 @@
+# babel-preset-flow
+
+> Babel preset for all Flow plugins.
+
+This preset includes the following plugins:
+
+- [transform-flow-strip-types](https://babeljs.io/docs/plugins/transform-flow-strip-types/)
+
+## Example
+
+**In**
+
+```javascript
```

Comment:
Not a huge deal, but probably don't need an example at the preset level (we don't have them for any of the others)?